test/functional/cmdLineTests/xlogTests/xlog.xml

```diff
@@ -29,7 +29,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 
 	<test id="-Xlog">
 		<command>$EXE$ $ENABLEXLOG$ -Xlog -version</command>
-		<output regex="no" type="success">openjdk version</output>
+		<output regex="yes" caseSensitive="no" javaUtilPattern="yes" type="success">(openjdk|semeru) version</output>
 		<output regex="no" type="success">java version</output>
```

Comment:
I think this should continue to be case-sensitive.
These two lines could be merged:
```
  <output regex="yes" javaUtilPattern="yes" type="success">(java|openjdk|semeru) version</output>
```
Likewise for all occurrences below.

Reply:
Merged checking for `java` & `openjdk` for both types `success` and `failure`.